news/2023/2023-05-10-skinning-contest-icons-of-an-era.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Skinning Contest: Icons of an Era"
+date: 2023-05-10 12:00:00 +0000
+---
+
+Celebrate 15 years of osu! by expressing your creativity through osu! skinning!
+
+![](https://assets.ppy.sh/contests/178/header.jpg)
+
+A lot has happened in the 15 years since osu!'s inception — and every time the community thought that we have reached the top, whether that'd be in mapping, skinning or gameplay, the boundaries of what is deemed possible kept getting pushed back further and further. So without further ado, let's go over the theme.
```

Comment:
a lot of words to say "boundaries have been pushed", but to me it doesn't seem a whole lot relevant to the contest nor skinning specifically.. ?

Reply:
It's something I wrote at the very inception of the news post so can probably revise it, it really only serves a purpose to fluff up the post and I thought it read nice.

I'll look to change it to something more relevant

Reply:
addressed in https://github.com/ppy/osu-wiki/pull/9335/commits/afeb1acb6cbe79e43ca246f5679ad582ff98f006 let me know if this is better